Assets/Scripts/Game/UserInterfaceWindows/DaggerfallGuildServicePopupWindow.cs

```diff
@@ -185,6 +189,14 @@ public override void OnPush()
             }
         }
 
+        public override void Update()
```

Comment:
What's this update needed for?

Reply:
Right now there's no mechanic in either Button or Panel to check and activate them (or their children) thru hotkeys, so I explicitly call my KeyboardActivation() methods explicitly.
I'm not satisfied with this solution, I'm sure it could be better integrated, but I don't know if that was already planned or how...

Reply:
Right now there's no mechanic in either Button or Panel to check and activate them (or their children) thru hotkeys, so I explicitly call my KeyboardActivation() methods from the different UI windows Update() methods.
I'm not satisfied with this solution, I'm sure it could be better integrated, but I don't know if that was already planned or how...